Tests/Packet++Test/Tests/SipSdpTests.cpp

```diff
@@ -648,7 +648,7 @@ PTF_TEST_CASE(SdpLayerParsingTest)
 } // SdpLayerParsingTest
 
 
-PTF_TEST_CASE(NotSdpLayerParseingTest)
+PTF_TEST_CASE(NotSdpLayerParsingTest)
```

Comment:
btw, you need to update here to run the test:
- Tests/Packet++Test/main.cpp
- TestDefinition.h

Reply:
Ok, I nailed it.